app/models/concerns/common_benefit_application.rb

```diff
@@ -27,7 +27,11 @@ def mailing_address
     addresses.where(mailing: true).first || NullAddress.new
   end
 
+  def stable_housing?
+    stable_housing == true || stable_housing == nil
```

Comment:
Why would stable_housing be true or nil?

Reply:
We didn't set a default value in the schema for it in `medicaid_applications`:

https://github.com/codeforamerica/michigan-benefits/blob/master/db/schema.rb#L171

Unlike for `snap_applications`, where we did:

https://github.com/codeforamerica/michigan-benefits/blob/master/db/schema.rb#L273

So during a Medicaid application, if the client never indicates they are homeless, the value is just never set to anything (which is `nil`). Since `nil` is falsey, that's why the checkbox was being filled on the 1426 PDF.

Reply:
Gotcha. I feel like we might want to update this to true in the controller for now, until we move toward a better way of reflecting state. So, that way we have a sure way of telling whether someone has answered affirmatively or whether they haven't answered at all. But, I don't think that this should block the feature getting out.